site/source/docs/getting_started/downloads.rst

```diff
@@ -135,8 +138,11 @@ Updating the SDK
 
 Type the following in a command prompt ::
 
-	# Fetch the latest registry of available tools.
-	./emsdk update
+	# Fetch the latest registry of available tools. (use `./emsdk update` if you did NOT install from git)
```

Comment:
comment here does not match the one for `git pull` in the previous appearance. I think the previous (`only if you used git clone to install`) is better?